app/components/ui/set-up-domain/progress-header/index.js

```diff
@@ -0,0 +1,42 @@
+// External dependencies
+import { bindHandlers } from 'react-bind-handlers';
+import i18n from 'i18n-calypso';
+import React, { PropTypes } from 'react';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+
+// Internal dependencies
+import ProgressBar from 'components/ui/progress-bar';
+import styles from './styles.scss';
+
+class ProgressHeader extends React.Component {
```

Comment:
It looks like this component can be transformed to arrow function style. The only refactoring requires is moving `redirect( 'myDomains' )` to the container like:
```js
{
    handleOnExitClick: () => redirect( 'myDomains' ) 
}
```

and we refactor from class definition to:
```js
const ProgressHeader = ( { content, progress } ) => {
...
}
```